pages/index.js

```diff
@@ -28,21 +30,30 @@ export default function Home({
             <Sorting onSorting={onSorting} />
           </StyledHeadingContainer>
           <StyledSection>
-            {entries.map((entry) => (
-              <Fragment key={entry.id}>
-                <BirthdayList
-                  id={entry.id}
-                  name={entry.name}
-                  birthday={entry.birthday}
-                  ideas={entry.ideas}
-                  notes={entry.notes}
-                  onUpdateEntry={onUpdateEntry}
-                  onDelete={onDelete}
-                  onMoreInfo={onMoreInfo}
-                  onUpdateEntryNotes={onUpdateEntryNotes}
-                />
-              </Fragment>
-            ))}
+            {entries.length === 0 ? (
+              <>
+                <StyledEmptyState>
+                  Es sind noch keine Geburtstage gespeichert...
+                </StyledEmptyState>
+                <Lottie animationData={Empty_BirthdayList} loop={true} />
```

Comment:
das ist bestimmt total süß, aber man kann es nicht sehen. Das Problem hatte ich auch. versuch mal in der Datei next.config.js folgendes:
```js
module.exports = {
  ...nextConfig,
  swcMinify: false,
  i18n: { locales: ["de"], defaultLocale: "de" },
};
```